### PR TITLE
Added historian script to archive chat messages

### PR DESCRIPTION
**Pull request by sohymg**

### CODE REVIEW

File: scripts/historian.coffee [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#
+# Commands:
+# No commands.
+#
+
+module.exports = (robot) ->
+  robot.hear /.*/i, (msg) ->
+    log = new Date() + ' | ' + msg.message.room + ' | ' + msg.message.user.name + ' | ' + msg.message
+    # msg.reply log
+    robot.http("http://192.168.99.100:56400")
```
Comment (fonglh) [inline scripts/historian.coffee]: Pull this out to a variable? Would be neater to edit if the IP/port changes.

Maybe put something like this at the top. See the waffles script.
`URL = process.env.HUBOT_SPOT_URL || "http://localhost:5051"`

File: scripts/historian.coffee [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    .header('Content-Type', 'application/text')
+    .post(log) (err, res, body) ->
```
Comment (fonglh) [inline scripts/historian.coffee]: Is the `(err, res, body) ->` supposed to be for the response? Can take out if you want to just ignore it?

Comment (sohymg) [inline scripts/historian.coffee]: take out meaning just `.post(log)` ? I am not familiar with coffee script

Comment (fonglh) [inline scripts/historian.coffee]: I think you need `.post(log)`, but maybe can drop the `(err, res, body) ->` part as that's another function.

File: scripts/historian.coffee (hunk 1 above)
Comment (sohymg) [inline scripts/historian.coffee]: Fixed. Does Github send out an email when I made a new commit?

**Comment by fonglh** [comment]
Looks good. Squash the commits then I'll merge.

**Comment by sohymg** [comment]
Squashed

**Pull request merged**